docker-compose.yml

```diff
@@ -68,15 +68,19 @@ services:
     depends_on:
       opi-intel-server:
         condition: service_healthy
+      opi-gw-test:
+        condition: service_completed_successfully # to get at least one operation registered in jaeger
     command: sh -c 'curl -s "http://jaeger:16686/api/traces?service=opi-intel-bridge&lookback=20m&prettyPrint=true&limit=10" | grep operationName'
 
   opi-test:
     image: docker.io/namely/grpc-cli
     networks:
       - opi
     depends_on:
-      opi-intel-server:
```

Comment:
need to keep this dep..

Reply:
I think `opi-test` should depend on only other tests. And if those tests need `opi-intel-server`, they should specify it explicitly in their `depends_on` section.

Reply:
maybe... wasn't clear from this commit what your intentions are...

Reply:
well... I can get it back for other PRs, if you insist and remove them in separate PRs

Reply:
no need, explanation is clear